docs/docs/manual/grelfunctions.md

```diff
@@ -210,7 +210,7 @@ Returns an array of strings [ a, fragment, z ] where a is the substring within s
 
 If the omitFragment boolean is true, for example with `"internationalization".partition("nation", true)`, the fragment is not returned. The output is [ "inter", "alization" ].
 
-You can use regex for your fragment. The expresion `"abcdefgh".partition(/c.e/)` will output  [“abc”, "cde", defgh” ]. 
+You can use regex for your fragment. The expression `"abcdefgh".partition(/c.e/)` will output  ["abc", "cde", "fgh" ].
```

Comment:
The output is actually `[ "ab", "cde", "fgh" ]` as suggested by the user.

Also, as long as you're cleaning things up, I'd either change the previous sentence to include "You can use a string or a regex" or just drop it altogether (my preference) since it's already documented just above.